### PR TITLE
CSVS-95 Support of several Vault Server addresses was implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To start a server in Docker container you should:
     * `ZOOKEEPER_RETRY_DELAY` - a delay between unsuccessful connection attempt to zookeeper and repeated attempt
     * `ZOOKEEPER_ROOT_NODE` - a root node for keeping application data in zookeeper ("/cs_vault_server" by default)
     * `ZOOKEEPER_MASTER_LATCH_NODE` - a node for master latch keeping ("/cs_vault_server_latch" by default)
-    * `VAULT_ENDPOINT` - vault endpoint
+    * `VAULT_ENDPOINTS` - list of vault endpoints, which is separated by comma
```
Comment (placccebo) [inline README.md]: are separated

File: src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala [hunk 2]
```diff
@@ -45,7 +45,9 @@ class CloudStackVaultController(vaultService: VaultService,
   private val logger = LoggerFactory.getLogger(this.getClass)
   private val accountEntityName = "accounts"
   private val vmEntityName = "vms"
-  private val vaultApiPath = s"${vaultService.endpoint}${RequestPath.vaultRoot}"
+  private val vaultApiPathsString = vaultService.endpoints.map { endpoint =>
+    s"$endpoint${RequestPath.vaultRoot}"
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala:50]: use https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html for a path concatenation

File: src/test/scala/com/bwsw/cloudstack/vault/server/vault/util/VaultRestRequestCreatorTestSuite.scala [hunk 3]
```diff
@@ -21,25 +21,25 @@ package com.bwsw.cloudstack.vault.server.vault.util
...
+import com.bwsw.cloudstack.vault.server.mocks.MockConfig.vaultRestRequestExecutorSettings
 import com.bwsw.cloudstack.vault.server.vault.TestData
 import com.bwsw.cloudstack.vault.server.vault.entities.Policy
 import com.bwsw.cloudstack.vault.server.vault.entities.Token.TokenInitParameters
 import com.bwsw.cloudstack.vault.server.vault.util.exception.VaultFatalException
-import org.scalatest.FlatSpec
+import org.scalatest.{FlatSpec, PrivateMethodTester}
 
-class VaultRestRequestCreatorTestSuite extends FlatSpec with TestData with BaseTestSuite {
+class VaultRestRequestCreatorTestSuite extends FlatSpec with TestData with PrivateMethodTester with BaseTestSuite {
```
Comment (placccebo) [inline src/test/scala/com/bwsw/cloudstack/vault/server/vault/util/VaultRestRequestCreatorTestSuite.scala]: rename according to the class name

File: src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala (hunk 2 above)
Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala:50]: remove unused method "createMissingAccountTokenTag()"

**Review by placccebo — CHANGES_REQUESTED** [review]
fix warnings, please))

File: README.md (hunk 1 above)
Comment (MedvedevBW) [reply README.md]: ok

File: src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala:50]: ok, nice advice ;)

File: src/test/scala/com/bwsw/cloudstack/vault/server/vault/util/VaultRestRequestCreatorTestSuite.scala (hunk 3 above)
Comment (MedvedevBW) [reply src/test/scala/com/bwsw/cloudstack/vault/server/vault/util/VaultRestRequestCreatorTestSuite.scala]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**